apps/hash-graph/libs/api/src/rest/data_type.rs

```diff
@@ -566,8 +566,8 @@ where
     let data_type = patch_id_and_parse(&type_to_update, schema).map_err(|report| {
         tracing::error!(error=?report, "Couldn't patch schema and convert to Data Type");
         StatusCode::UNPROCESSABLE_ENTITY
-        // TODO - We should probably return more information to the client
-        //  https://app.asana.com/0/1201095311341924/1202574350052904/f
+        // @todo We should probably return more information to the client
+        // @see https://linear.app/hash/issue/H-3009
```

Comment:
Rust does not has a `@keyword` syntax. Typically, we use `TODO: ` (I don't know why it is `Todo - ` here).

Reply:
Also, the `see` should be indented to make it part of the TODO block.
```suggestion
        // TODO: We should probably return more information to the client
        //   see https://linear.app/hash/issue/H-3009
```

Reply:
I was hoping we could have a standardized way repo-wide of searching for todos in the codebase.

Does the difference in formatting result in any loss of functionality in Rust IDEs (or other benefits, other than language norm-conformance), or can we agree to internally standardize on `@todo` across ecosystems?

Reply:
If `@see` is not indented, it's neither highlighted, nor listed in the TODO overview in JetBrains IDEs:
<img width="564" alt="image" src="https://github.com/hashintel/hash/assets/21277928/8f3a58d7-e96a-468d-9959-154adbc36e06">
<img width="548" alt="image" src="https://github.com/hashintel/hash/assets/21277928/1ffb80a9-0173-4be6-b2eb-8dc060f013a3">

The `@` sign typically is used for documentation purposes and documentation in code differs between languages, Rust for example uses Markdown for documentation